src/wallet-web-docs/Main.hs

```diff
@@ -3,6 +3,7 @@ import           Universum
 import           Pos.Wallet.Web (walletDocsText)
 
 main :: IO ()
-main = writeFile fp walletDocsText >> putStrLn ("See " <> fp)
-  where
-    fp = "wallet-web-api.md"
+main = do
```

Comment:
I left this since I was thinking of adding another generated document, but I can fix it for now. Fixed.